scripts/build.mjs

```diff
@@ -78,10 +55,12 @@ async function main() {
       {
         ...pkg,
         files: ["dist-*/**", "bin/**"],
-        main: "dist-node/index.js",
-        module: "dist-web/index.js",
-        types: "dist-types/index.d.ts",
-        source: "dist-src/index.js",
+        exports: {
+          ".": {
+            types: "./dist-types/index.d.ts",
+            import: "./dist-bundle/index.js",
```

Comment:
You'll need to add `default` here as some tooling doesn't seem to be able to import the `import`

See https://github.com/octokit/core.js/issues/667#issuecomment-2037592361